app/models/country.rb

```diff
@@ -7,7 +7,7 @@ class Country < ApplicationRecord
   has_many :sub_locations
   has_many :designations, -> { uniq }, through: :protected_areas
 
-  belongs_to :parent, class_name: "Country", foreign_key: :country_id
+  belongs_to :parent, class_name: "Country", foreign_key: :country_id, optional: true
```

Comment:
Could you please clarify why this became now optional?

Reply:
Ah sure, so sometimes this can be `nil` and Rails 5 expects belongs_to not to be `nil`. I think that is correct @benkt2 ? There was an article about this: "Rails 5 makes belongs_to association required by default" https://blog.bigbinary.com/2016/02/15/rails-5-makes-belong-to-association-required-by-default.html